contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -847,11 +848,12 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, C
             previous_contract: contract_info.contract,
         });
     }
-    if contract_info.version == "0.12.1" {
+    let contract_version: Version = contract_info.version.parse()?;
+    let current_version: Version = CONTRACT_VERSION.parse()?;
+    if contract_version < current_version {
```

Comment:
Contract version upgrade should start from the missing version.
Example: if current_version is v0.14.0 and contract version is v0.13.0, it should skip 0.12 version.